js/navigation.js

```diff
@@ -0,0 +1,27 @@
+// Navbar
+const menuOpen = document.getElementById('menuOpen');
+const links = document.querySelectorAll('.header__navbar__list__item a');
+
+const toggleMenu = () => {
+  const navbar = document.querySelector('.header__navbar__list__container');
+  navbar.classList.toggle('showNavbar');
+
+  if (window.matchMedia('(max-width: 767px)').matches) {
+    if (navbar.classList.contains('showNavbar')) {
+      menuOpen.classList.add('fa-times');
+      menuOpen.classList.remove('fa-bars');
+      navbar.style.left = '0';
+      document.body.style.overflow = 'hidden';
+    } else {
+      menuOpen.classList.remove('fa-times');
+      menuOpen.classList.add('fa-bars');
+      document.body.style.overflow = 'scroll';
+      navbar.style.left = '-100%';
+    }
+  }
+};
+menuOpen.addEventListener('click', toggleMenu);
```

Comment:
- Hi, the requirement says when any of the links are clicked, the main area changes to display only one of the three content sections. I would advise you to follow the requirements as stated. Here is the wireframe.
![Capture](https://github.com/Feliperto/AwesomeBooks/assets/63972670/3145aa79-082a-48a1-8ae0-68c4ada7ec26)